src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1095,8 +1138,18 @@ async def verify_identification_code(
     )
     try:
         privacy_request.verify_identity(db, provided_code.code)
+        policy: Optional[Policy] = Policy.get(
+            db=db, object_id=privacy_request.policy_id
+        )
+        if config.notifications.send_request_receipt_notification:
+            _send_privacy_request_receipt_email_to_user(
+                db, policy, privacy_request.get_persisted_identity().email
+            )
     except IdentityVerificationException as exc:
         raise HTTPException(status_code=HTTP_400_BAD_REQUEST, detail=exc.message)
+    except EmailDispatchException as exc:
+        # not fatal to request lifecycle, do not raise error, continue with request
+        logger.info("Email dispatch failed with exception %s", Pii(exc))
```

Comment:
Similarly, instead of wrapping this exception in Pii, I'd fix the one example of the raised exception that could potentiallly contain Pii.